tests/dummy/app/templates/application.hbs

```diff
@@ -22,7 +22,13 @@
       <p>Since there is no way to automate screen readers for testing, this is an interactive script to help you run all of the tests inside of each screen reader.</p>
 
       <h2>Tests</h2>
-      <p>Each of these tests should be reviewed in every single OS/AT/Browser combination.</p>
+      <p>Each of these tests should be reviewed in the appropriate OS/AT/Browser combination. Common browser/AT combinations include:</p>
+      <ul role="list">
+        <li role="listitem">Chrome + ChromeVox (Chromebook)</li>
```

Comment:
we might not need to- that is sort of automatic for me. I was more focused on the content. Actually I have a list of this stuff let me double check rq.